gamearea.py

```diff
@@ -201,10 +204,14 @@ def __motion_cb(self, widget, event):
                 elif y + height > alloc.height - 25:
                     y = alloc.height - height - 25
 
-                if x < left_limit and x + width > left_limit and self.clicked[0] < alloc.width // 2:
+                if x < left_limit \
+                        and x + width > left_limit \
+                        and self.clicked[0] < alloc.width // 2:
```

Comment:
I wonder if using parentheses her would make this more readable and rid the line continuation character.

Reply:
Yes, did that.